AuthenticatorShared/UI/Platform/Settings/Settings/SettingsViewTests.swift

```diff
@@ -110,6 +121,15 @@ class SettingsViewTests: AuthenticatorTestCase {
         )
     }
 
+    /// Tests the view renders correctly.
+    func test_viewRenderWithBiometricsAvailable() {
```

Comment:
🤔 I'm surprised there were no changed snapshots, but I guess I'd just missed doing ones with biometrics available to begin with.

Reply:
I think there were previews in the View, but this is one where we didn't just snapshot all previews. So they weren't previously getting a snapshot.